vmc/resource_vmc_cluster.go

```diff
@@ -100,14 +84,7 @@ func clusterSchema() map[string]*schema.Schema {
 			Optional:    true,
 			Description: "The instance type for the esx hosts added to this cluster.",
 			ValidateFunc: validation.StringInSlice(
-				[]string{constants.HostInstancetypeI3, constants.HostInstancetypeR5, constants.HostInstancetypeI3EN, constants.HostInstancetypeI4I}, false),
-		},
-		"storage_capacity": {
```

Comment:
You have to document in CHANGELOG.md that this is a breaking change for the Cluster and SDDC schema.

Reply:
Added notes in the CHANGELOG.md for the next future release which should be 1.14.0